public/controllers/agent/agents-preview.js

```diff
@@ -80,6 +80,10 @@ export class AgentsPreviewController {
     this.$scope.$broadcast('wazuhFilter', { filter });
   }
 
+  query(query, search) {
+    this.$scope.$broadcast('wazuhQuery', { query, search });
+  }
+
   showAgent(agent) {
```

Comment:
We may want to remove unused functions? I don't know but we must review it because we've removed some parts of our HTML view.